packages/tailwind/tailwind-base.cjs

```diff
@@ -122,6 +255,24 @@ module.exports = (options = {}) => {
           '.h4': {
             [h4]: {},
           },
+          '.h5': {
+            [h5]: {},
+          },
+          '.h6': {
+            [h6]: {},
+          },
+          '.body': {
+            [body]: {},
+          },
+          '.lead': {
+            [lead]: {},
+          },
+          '.blockquote': {
+            [blockquote]: {},
+          },
+          '.description': {
```

Comment:
Skal denne kun brukes på bilder? Er da for eksempel `caption` mer beskrivende enn description, for å ikke forvirre med `description` brukt i skjemaer.

Reply:
Det er et godt poeng. Må dobbeltsjekke med Kristian, men det han sa var at det skulle brukes til bilder. Er litt usikker på akkurat `caption`, selv om det er et bra navn så er det kanskje lett å forveksle med `<caption>` som man bruker i tabeller?

Reply:
Har tatt en lang diskusjon med Kristian nå, og det viser seg at vi skal ha samme font på denne og vår eksisterende `<Description/>`. Så har dratt inn denne nye `description`-klassen inn i den komponenten, slik at de samsvarer. Kristian landet også på at font-light utgår for `<Description/>`.